geonode/local_settings.py.qgis.sample

```diff
@@ -44,14 +49,16 @@ if 'geonode.geoserver' in INSTALLED_APPS:
 if 'geonode.geoserver' in GEONODE_APPS:
     GEONODE_APPS.remove("geonode.geoserver")
 
-if 'LOCAL_GEOSERVER' in locals():
-    MAP_BASELAYERS.remove(LOCAL_GEOSERVER)
-    del LOCAL_GEOSERVER
-
-if 'PUBLIC_GEOSERVER' in locals():
+if 'PUBLIC_GEOSERVER' in locals() and \
```

Comment:
Fix syntax of excluding Geoserver basemap if we use QGIS Server backend (because GeoServer doesn't exists)